cap/modules/schemas/utils.py

```diff
@@ -104,3 +112,27 @@ def is_later_version(version1, version2):
                 return False
             elif patch1 == patch2:
                 return False
+
+
+def validate_schema_config(config):
+    configs = ['notifications', 'reviewable', 'repositories']
```

Comment:
no need to check for each one differently, we can have one schema extended/combined with all